guide/daily-spending-wallet/receiving.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: guide
+title: Receiving bitcoin
+description: How receiving works for a daily spending bitcoin wallet.
+nav_order: 2
+parent: Daily spending wallet
+permalink: /guide/daily-spending-wallet/receiving/
+image: /assets/images/guide/daily-spending-wallet/requesting/request-banner-preview(1200x630).jpg
+image_base: /assets/images/guide/daily-spending-wallet/receiving/
+redirect_from:
+ - /guide/payments/receive/
+ - /guide/daily-spending-wallet/funding/
+ - /guide/onboarding/funding-a-wallet/
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+    image = "/assets/images/guide/daily-spending-wallet/requesting/request-banner(1600x600).png"
+    retina = "/assets/images/guide/daily-spending-wallet/requesting/request-banner(3200x1200).png"
+    mobile = "/assets/images/guide/daily-spending-wallet/requesting/request-banner(400x200).png"
+    mobileRetina = "/assets/images/guide/daily-spending-wallet/requesting/request-banner(800x400).png"
+    alt-text = "Image: Text balloon containing a bitcoin."
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+<!--
+
+Illustration source
+
+- https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/Design-Guide-Bosch?node-id=201%3A687
+
+-->
+
+# Receiving bitcoin
+
+Once a user has shared a payment request, the next step is receiving the payment. Ideally, there is little to no actions for a user to take at this stage of the payment flow. 
```

Comment:
`payment request` could link to the requesting page. If someone lands on this page directly, they may want to read up on that context.